spec/Powertrain/Battery.vspec

```diff
@@ -145,7 +145,7 @@ Charging.Mode:
 Charging.Status:
```

Comment:
* @danielwilms will check internally (@adobekan, or you?) 
* other proposal: `isCharging` instead of `Status`